supabase-ui/package-lock.json

```diff
@@ -1237,7 +1237,10 @@
       "version": "7.12.5",
       "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.12.5.tgz",
       "integrity": "sha512-plcc+hbExy3McchJCEQG3knOsuh3HH+Prx1P6cLIkET/0dLuQDEnrT+s27Axgc9bqfsmNUNHfscgMUdBpC9xfg==",
+<<<<<<< HEAD
```

Comment:
there are merge conflicts in this file. You should remove it, run `npm install` to re-create it @MildTomato